pkg/config/generate.go

```diff
@@ -316,6 +312,17 @@ func Generate(cs *acsapi.ContainerService, c *Config) (err error) {
 			key:         &c.Certificates.Registry.Key,
 			cert:        &c.Certificates.Registry.Cert,
 		},
+		// Openshift Console is BYO type of certificate. In a long run we should enable users to configure their own certificates. For this reason we decouple it from all OCP certs and make it self-sign
```

Comment:
s/In a long run/In the long run/

Also, can you break this comment down to two or three separate lines?

Reply:
Done :) on my screen it takes only 1/3 of the screen :D